main/ertp/guide/amount-math.md

```diff
@@ -10,111 +10,88 @@ uses `amountMath` methods for all these operations.
 `amountMath` methods also check their arguments' `brand`, 
 throwing an error if the wrong `brand` was used.
 
-An `amountMath` is one of three different kinds, each of which
+An `amountMath` is one of two kinds, each of which
```

Comment:
There's no such thing as "an amountMath" since it's a library. Let's say something like the amountMath library functions work for both fungible and nonfungible tokens.